fish-rust/src/wchar_ffi.rs

```diff
@@ -143,3 +143,15 @@ impl WCharFromFFI<WString> for cxx::SharedPtr<cxx::CxxWString> {
         WString::from_chars(self.as_chars())
     }
 }
+
+impl WCharFromFFI<Vec<u8>> for cxx::UniquePtr<cxx::CxxString> {
```

Comment:
Nitpick: please use `WCharFromFfi` to use conventional PascalCase (not all-caps FFI).

Reply:
The trait is currently named `WCharFromFFI` and not `WCharFromFfi`, are you suggesting that I rename the entire trait?

Reply:
Yes

Reply:
Sorry, I thought you added the trait. No, please leave the existing trais as-is and ignore this for now! :)